app/renderer/components/styles/global.js

```diff
@@ -327,6 +328,13 @@ const globalStyles = {
         background: '#ccc'
       }
     }
+  },
+
+  // TODO (Suguru): move them to payment.js after style refactoring is done
```

Comment:
I see. I'm going to address that on another PR (as it is out of scope of this PR and it'll take some time).